Issue/3668 rotate wpadmin

nbradbury opened the pull request:
Resolved #3668 by:

1. Having WebViewActivity save & restore the webView state, which retains the current URL
2. Only loading the desired URL on initial creation (ie: when `savedInstanceState` is null) so it doesn't reload upon rotation

Note that this problem wasn't specific to "View Admin" - it happened everywhere a WebViewActivity was used.

maxme commented:
Looks good, but there is something weird on Calypso pages, after rotation the screen is stuck on the empty page logo (loading).

<img width="590" alt="screen shot 2016-01-29 at 13 59 53" src="https://cloud.githubusercontent.com/assets/40213/12676249/db8c85f6-c690-11e5-8f1d-a3dacecd3816.png">

nbradbury commented:
@maxme Glad you caught that problem reloading Calypso pages! Turns out that even when restoring the webView state, the previous settings (incuding WebViewClient and WebChromeClient) still need to be applied. So, I updated WPWebViewActivity with the necessary corrections.

These corrections required updating SSLCertsViewActivity and LicensesActivity, since they implement WPWebViewActivity.

In addition, I noticed that "View Site" suffered from the same loss of navigation after rotation, so I updated it to implement WPWebViewActivity.

TL;DR Ready for another look.

maxme commented:
Looks good! :shipit:
